website/pages/docs/connect/terminating_gateway.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Connect - Terminating Gateways
+sidebar_title: Terminating Gateways
+description: >-
+  A terminating gateway enables traffic from services in the Consul
+  service mesh to services outside the mesh. This section details
+  how to configure and run a terminating gateway.
+---
+
+# Terminating Gateways
+
+-> **1.8.0+:**  This feature is available in Consul versions 1.8.0 and newer.
+
+Terminating gateways enable connections from services in the Consul service mesh to
+services outside the mesh. These gateways effectively act as Connect proxies that can
+represent more than one service. They terminate Connect mTLS connections, enforce intentions,
+and forward requests to the appropriate destination.
+
+## Security Considerations
+
+We recommend that terminating gateways not be exposed to the WAN or open internet. This is because terminating gateways
+hold certificates to decrypt Consul Connect traffic directed at them and may be configured with credentials to connect
+to linked services. Connections over the WAN or open internet should flow through [mesh gateways](/docs/connect/mesh_gateway)
+whenever possible since they are not capable of decrypting traffic or connecting directly to services.
+
+By specifying a path to a [CA file](/docs/agent/config-entries/terminating-gateway#cafile) connections
+from the gateway will be encrypted using one-way TLS authentication. If a path to a
+[client certificate](/docs/agent/config-entries/terminating-gateway#certfile)
+and [private key](/docs/agent/config-entries/terminating-gateway#keyfile) are also specified connections
+from the gateway will be encrypted using mutual TLS authentication.
+
+If none of these are provided, Consul will **only** encrypt connections to the gateway and not
+from the gateway to the destination service.
+
+-> **Note:** If certificates and keys are configured the gateway will upgrade HTTP connections to TLS. 
+Client applications can issue plain HTTP requests even when connecting to servers that require HTTPS.
+
+## Prerequisites
+
+Each terminating gateway needs:
+
+1. A local Consul agent to manage its configuration.
+2. General network connectivity to services within its local Consul datacenter.
+
+Terminating gateways also require that your Consul datacenters are configured correctly:
+
+- You'll need to use Consul version 1.8.0.
+- Consul [Connect](/docs/agent/options#connect) must be enabled on the datacenter's Consul servers.
+- [gRPC](/docs/agent/options#grpc_port) must be enabled on all client agents.
+- [Centralized configuration](/docs/agent/options#enable_central_service_config) must be enabled.
```

Comment:
I don't think this was a prereq in the docs for ingress gateways, should it be?

Reply:
It doesn't need to be actually. Fixed.